web/src/components/home/subcomponents/welcome/welcome.js

```diff
@@ -1,5 +1,40 @@
+import { Box, Button, Stack } from "@mui/material";
+import welcomePic from "assets/images/teacher-demo-2-blurred-min.JPG";
+import welcomeStyles from "./welcome.styles.js";
+import FeatureFlags from "infra/feature_flags";
+
 function Welcome() {
-  return <div>Welcome</div>;
+  return (
+    <Stack sx={welcomeStyles.sectionLayout}>
+      <Stack justifyContent="center" sx={welcomeStyles.titleBox}>
+        <Box>
+          <Box sx={welcomeStyles.subtitle}>Discover your rhythm at</Box>
```

Comment:
Should we wrap text into Typography? (if it's not in a component like button)

Reply:
Good call, updated